src/main/scala/com/scalapenos/riak/RiakBucket.scala

```diff
@@ -70,5 +70,9 @@ trait RiakBucket {
   def lastWriteWins_=(value: Boolean): Future[Unit] = setLastWriteWins(value)
   def setLastWriteWins(value: Boolean): Future[Unit] = setProperties(Set(LastWriteWins(value)))
 
-}
+  // ============================================================================
+  // Special operations (Use with care)
+  // ============================================================================
 
+  def allKeys(): Future[RiakKeys]
```

Comment:
No, I like the idea! Let me know what you think of the implementation. Right now `UnsafeBucketOperations` extends `RiakBucket`, because I think in practice it is more handy.